test/jdk/sun/net/www/http/KeepAliveCache/B8293562.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2005, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Daniel, since this is a new file, the copyright year would just be `2022`.